### PR TITLE
New C, CPP, SQLC and SQLCPP object types plus multiple compile options

### PR DESCRIPTION
**Pull request by sbestgen**
Adding new types plus compiles that specify more than one compile option to process. Edmund, I've tested this on my system but it might be worth trying on another box as well?

### CODE REVIEW

File: QCPPSRC/Rules.mk [hunk 1]
```diff
@@ -0,0 +1,6 @@
+MODULEs := PWD.MODULE 
+PGMs :=  PWD.PGM
```
Comment (edmundreinhardt) [inline QCPPSRC/Rules.mk:2]: Don't need these 2 lines any more

File: QCSRC/Rules.mk [hunk 2]
```diff
@@ -0,0 +1,11 @@
+MODULEs := PUTIFSCHK.MODULE
```
Comment (edmundreinhardt) [inline QCSRC/Rules.mk:1]: drop top 2 lines (MODULES and PGMS)

File: QSQLCPPSRC/Rules.mk [hunk 3]
```diff
@@ -0,0 +1,7 @@
+MODULEs := ANZ_FILE.MODULE
```
Comment (edmundreinhardt) [inline QSQLCPPSRC/Rules.mk:1]: drop MODULEs PGMs

File: QSQLCSRC/Rules.mk [hunk 4]
```diff
@@ -0,0 +1,9 @@
+MODULEs := ANZ_FILE2.MODULE
```
Comment (edmundreinhardt) [inline QSQLCSRC/Rules.mk:1]: drop MODULEs PGMs

**Review by edmundreinhardt — CHANGES_REQUESTED** [review]

File: QCPPSRC/Rules.mk (hunk 1 above)
Comment (sbestgen) [reply QCPPSRC/Rules.mk]: New code committed.

File: QCSRC/Rules.mk (hunk 2 above)
Comment (sbestgen) [reply QCSRC/Rules.mk]: New code committed.

File: QSQLCPPSRC/Rules.mk (hunk 3 above)
Comment (sbestgen) [reply QSQLCPPSRC/Rules.mk]: New code committed.

File: QSQLCSRC/Rules.mk (hunk 4 above)
Comment (sbestgen) [reply QSQLCSRC/Rules.mk]: New code committed.

**Pull request merged**